department-of-veterans-affairs/va.gov-cms
# VACMS-8274 clean up media menu

## Description
Small note: the default media tab says 'table' and neither @JayDarnell nor I could figure out why. I also combined the media - images view with the main media view.

Closes #8274 .

## Testing done
Logged in and confirmed only one media link exists in admin menu and related views are nested under it as either links or tabs. 

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/157528429-b743b69d-bc28-4219-a2db-697572de83a3.png)

![image](https://user-images.githubusercontent.com/8375335/157528571-f6634041-9217-4af4-8ea8-c2a299b74f3b.png)


## QA steps

As a cms user I can
1. open the 'content' menu and hover over the media item
2. click on `Media` and see the new tabs
3. Then validate Acceptance Criteria from issue
- [ ] We should have only one menu item under Content with the word "media"
- [ ] Add media links and Files link should be children of the media menu
- [ ] Media Images, Media PDFs, Bulk Edit should be tabs on the media view

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
